src/developer_config.yaml

```diff
@@ -2,15 +2,14 @@ global:
   log_to_file: True # Write logs to .log file
   logging_level: "DEBUG"
   table_config: "SingleLine"
-  validate_postcodes: True
```

Comment:
Why has this been removed? Is it a mistake?

Reply:
This config would skip the entire postcodes module. We need this to run to get the harmonised postcodes for use later in the pipeline. I think we only had this setting since it used to take really long to run.

I think we should maybe also remove the "postcode_csv_check" setting since it returns an empty list when false. What do you recommend?